client/main.lua

```diff
@@ -1007,3 +1007,733 @@ end)
 AddEventHandler('esx:onPlayerSpawn', function(spawn)
 	isDead = false
 end)
+local HasAlreadyEnteredMarker, LastZone = false, nil
```

Comment:
Create a new client.lua file for this. This will cause error on script because of all the ESX reinitialization.

Reply:
If I'm not mistaken po sir this is an old commit, the new commit for client/main.lua is only 1004 lines long po.